en/identity-server/7.1.0/docs/references/service-extensions/in-flow-extensions/custom-authentication/api/custom-authentication-action-v1.yaml

```diff
@@ -434,8 +438,8 @@ components:
           format: uri
           description: Claim uri defined in the 'http://wso2.org/claims' dialect
         value:
-          type: string
-          description: Claim value
+          type: object
+          description: Claim value. If the claim is multi-valued, it should be a list of strings (the `MultiAttributeSeparator` is not allowed in claim values). Otherwise, it should be a single string.
```

Comment:
Here also let's say 'Note that the value configured for 'MultiAttributeSeparator' is not allowed in claim values'